### PR TITLE
Update licence

### PR DESCRIPTION
**Pull request by tgrippa**

### CODE REVIEW

**Comment by tgrippa** [comment]
Hi Moritz, can you merge the pull request ? Thanks

File: SRC/walous_obia_modeltuning.pbs [hunk 1]
```diff
@@ -1,5 +1,31 @@
 #!/bin/bash -l
 
+"""
```
Comment (mlennert) [inline SRC/walous_obia_modeltuning.pbs]: This is a bash file, so you cannot use the """ python block comment format. Instead, each line has to be preceded by a #.

File: SRC/walous_obia_tiles_data_creation.pbs [hunk 2]
```diff
@@ -1,5 +1,31 @@
 #!/bin/bash -l
 
+"""
```
Comment (mlennert) [inline SRC/walous_obia_tiles_data_creation.pbs]: idem (#, not """)

File: SRC/walous_obia_trainingfile_creation.pbs [hunk 3]
```diff
@@ -1,5 +1,31 @@
 #!/bin/bash -l
 
+"""
```
Comment (mlennert) [inline SRC/walous_obia_trainingfile_creation.pbs]: #, not """

File: LICENSE [hunk 4]
```diff
@@ -1,622 +1,387 @@
...
+			
+	
+WALOUS_OCS_OBIA software has been developed by:
+=====================================
+
+Lennert Moritz (Université Libre de Bruxelles)
+
+...
+
+The author is indebted to Grippa Taïs (Université Libre de Bruxelles), Radoux Julien (Université catholique de Louvain) and Beaumont Benjamin (Institut Scientifique de Service Public) for their guidance.
```
Comment (mlennert) [inline LICENSE]:
```suggestion
The author is indebted to Grippa Taïs (Université Libre de Bruxelles), Radoux Julien (Université catholique de Louvain) and Beaumont Benjamin (Institut Scientifique de Service Public) for their support.
```

**Review by mlennert — CHANGES_REQUESTED** [review]

**Comment by tgrippa** [comment]
Hi @mlennert, the requested changes have been made.

**Pull request merged**